pkg/controller/import-controller.go

```diff
@@ -93,20 +99,20 @@ func (ic *ImportController) processPvcItem(pvc *v1.PersistentVolumeClaim) error
 		needsSync = false
 	}
 	if pod == nil && needsSync {
-		ep, err := getEndpoint(pvc)
+		podEnvVar.ep, err = getEndpoint(pvc)
```

Comment:
Wondering about the use of a global variable here.  The controller would have to be single threaded for this to be safe.  Which I believe would be the case if threadiness is > 1 here:  https://github.com/kubevirt/containerized-data-importer/blob/master/pkg/controller/controller.go#L170

Reply:
You are right, that isn't safe, we should create a new struct here and pass it down instead of using a global.

Reply:
Thanks! Forgot this can run in a multi-threaded environment. I am going to declare the variable not global. inside the if block.I believe it satisfies this.

Reply:
how to we change this after the merge? Can we reopen this? or I need a new PR?

Reply:
Just create a new PR @zvikorn that fixes the issue.